settings.gradle.kts

```diff
@@ -0,0 +1,22 @@
+rootProject.name = "djl"
+
+val reserved = listOf("gradle", "buildSrc")
```

Comment:
I prefer list all project here, it can also treat as an project index file.

Reply:
> I prefer list all project here

Ok, as you wish

> it can also treat as an project index file.

What do you mean, exactly?